pkg/workloads/cortex/serve/nginx.conf.j2

```diff
@@ -0,0 +1,101 @@
+# Copyright 2020 Cortex Labs, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# good articles to read
+# https://hub.packtpub.com/fine-tune-nginx-configufine-tune-nginx-configurationfine-tune-nginx-configurationratio/
+# https://www.nginx.com/blog/tuning-nginx/
+# https://www.digitalocean.com/community/tutorials/understanding-nginx-http-proxying-load-balancing-buffering-and-caching
+# https://serverfault.com/a/788703
+# https://stackoverflow.com/questions/59846238/guide-on-how-to-use-regex-in-nginx-location-block-section
+
+daemon off;
+# maximum number of open files per worker
+worker_rlimit_nofile 65535;
+worker_processes 1;
+
+thread_pool pool threads={{ CORTEX_PROCESSES_PER_REPLICA | int }};
+
+events {
+    # max_clients = (worker_processes * worker_connections ) / (X * 2) for reverse proxy
+    # max clients is also limited by the number of socket connections available on the system (~64k)
+    worker_connections 65535;
```

Comment:
Would it work to set `worker_connections` to `CORTEX_MAX_REPLICA_CONCURRENCY` instead of using `limit_conn_zone` + `limit_conn`?